docs/DeveloperGuide.adoc

```diff
@@ -798,17 +800,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *`|new user |see usage instructions |refer to instructions when I forget how to use the App
+
+|`* * *`|user |add a new person | include employees that have just entered the company
+
+|`* * *`|user |each employee has his/her dedicated timetable | trace progress of that person
+
+|`* * *`|user |add a timetable entry on anyone's timetable | assign jobs to employees
+
+|`* * *`|user |edit a timetable entry |update timetable entry information
+
+|`* * *`|user |delete a timetable entry | remove a job or event that is canceled
+
+|`* * *`|user |give employees a rating |evaluate their performance
+
+|`* * *`|user |change employees' rating | update my rarting record when the performance of an employee changes
+
+|`* * *`|user |add notes on a timetable entry or on a person | include more details about the entry
+
+|`* * *`|user |search employees by specific criteria | search employees more easily
```

Comment:
Might be a good idea to enlist a couple of criteria's possible. That way reader can understand which criteria you are planning to consider.